custom/opm/reports.py

```diff
@@ -1447,30 +1453,33 @@ def rows(self):
         ret = super(HealthMapSource, self).rows
         new_rows = []
         for row in ret:
-            awc = row[0]
+            awc = row[1]
             awc_map = gps_mapping.get(awc, None) or ""
-            gps = awc_map["gps"] if awc_map else "--"
             extra_columns = ["--"] * 4
             if awc_map:
                 extra_columns = []
                 for key in ["gps", "AWW", "Block", "GP"]:
                     extra_columns.append(awc_map.get(key, "--"))
-            escaped_row = [row[0]]
-            for cell in row[1:]:
+            escaped_row = ['', row[1]]
+            for cell in row[2:]:
                 # _unformat_row([<html>] => ["N - f%"])
-                percent = re.findall(pattern, _unformat_row([cell])[0])
-                html_cell = {"html": cell, "sort_key": int(percent[0] if percent else 0)}
+                try:
+                    percent = re.findall(pattern, _unformat_row([cell])[0])
```

Comment:
Does this still work the way it worked? Because, new HSR outputs numbers in different formats not the HTML format like old HSR used to.